<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingSettingsDialog</class>
 <widget class="QDialog" name="ProcessingSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Processing Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;aaa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">a
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="BSTab">
    <attribute name="title">
     <string>BackGround Subtraction</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Number of iterations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BSIterationsEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>[1-999]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resetDilateToDefaultsButton">
        <property name="text">
         <string>Reset to Defaults</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="ColorTab">
    <attribute name="title">
     <string>Color</string>
    </attribute>
    <widget class="QPushButton" name="resetColorToDefaultsButton">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>286</y>
       <width>135</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset to Defaults</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ColorRGBButton">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>47</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RGB</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ColorTypeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="ColorHSVButton">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>74</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ColorTypeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="ColorYCrCbButton">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>101</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>YCrCb</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ColorTypeGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>13</y>
       <width>59</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Parameters:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ColorParam1Label">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>48</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ColorParam1Edit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="ColorParam2Label">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>84</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ColorParam2Label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ColorParam2Edit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="ColorParam3Label">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>120</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ColorParam3Edit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="ColorParam1Label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ColorParam1Edit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="ColorParam2Edit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="ColorParam3Label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ColorParam3Edit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>249</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>249</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="okCancelBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetAllToDefaultsButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset All to Defaults</string>
   </property>
  </widget>
  <zorder>okCancelBox</zorder>
  <zorder>applyButton</zorder>
  <zorder>resetAllToDefaultsButton</zorder>
  <zorder>line_1</zorder>
  <zorder>line_2</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>ColorRGBButton</tabstop>
  <tabstop>ColorHSVButton</tabstop>
  <tabstop>ColorYCrCbButton</tabstop>
  <tabstop>ColorParam1Edit</tabstop>
  <tabstop>ColorParam2Edit</tabstop>
  <tabstop>ColorParam3Edit</tabstop>
  <tabstop>resetColorToDefaultsButton</tabstop>
  <tabstop>BSIterationsEdit</tabstop>
  <tabstop>resetDilateToDefaultsButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>resetAllToDefaultsButton</tabstop>
  <tabstop>okCancelBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelBox</sender>
   <signal>accepted()</signal>
   <receiver>ProcessingSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelBox</sender>
   <signal>rejected()</signal>
   <receiver>ProcessingSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ColorTypeGroup"/>
 </buttongroups>
</ui>
